src/export/option.ts

```diff
@@ -234,4 +250,17 @@ export interface EChartsOption extends ECBasicOption {
 
     options?: EChartsOption[];
     baseOption?: EChartsOption;
-}
\ No newline at end of file
+}
+
+export {
+    TooltipFormatterCallback as TooltipComponentFormatterCallback,
+    TopLevelFormatterParams as TooltipComponentFormatterParams,
+    LabelFormatterCallback,
+    CallbackDataParams as DefaultLabelFormatterParams,
+    AnimationDurationCallback,
+    AnimationDelayCallback,
+    AnimationDelayCallbackParam as AnimationDelayCallbackParams,
+    LabelLayoutOptionCallbackParams,
+    LabelLayoutOptionCallback,
+    PositionCallback
```

Comment:
`PositionCallback` is only used in the tooltip. So it can also be TooltipComponentPositionCallback